typescript/best-practice-web/test/specs/visual.spec.ts

```diff
@@ -0,0 +1,20 @@
+import loginPage from '../pages/login.page';
+import {LOGIN_USERS} from '../e2eConstants'
+import cartPage from '../pages/cart.page';
+import inventoryPage from '../pages/inventory.page';
+
+describe('My web app', () => {
+    it('should look correct on desktop web', () => {
+        loginPage.open();
+        browser.execute('/*@visual.init*/', 'Sauce Demo App');
```

Comment:
Can you explain in a comment what you are doing here? That would make it better understandable for the users